week2/exercise/Program.cs

```diff
@@ -0,0 +1,131 @@
+
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello fuckingWorld!");
+/*
+1. Caclulator
+
+Make a GET endpoint that will take parameter number1 and number2 and based on operation parameter will perform 
+one of following operations: addition, substraction, multiplication. If number1 or number2 are not a number return bad request response. 
+For operation valid values are add, substract, multiplay.
+
+Example: GET /calculate?number1=5&number2=10&operation=add would return 15 as a result.
+*/
+
+app.MapGet("/calculator", (string numberOne, string numberTwo, string operation) =>
+{
+    var numberOneIsValid = int.TryParse(numberOne, out var parseNumberOne);
+    var numberTwoIsValid = int.TryParse(numberTwo, out var parseNumberTwo);
+
+    if (!numberOneIsValid || !numberTwoIsValid)
+    {
+       return Results.BadRequest("One of the numbers is not valid");
+       // Console.WriteLine("not valid numbers");
+    }
+
+    int result = 0;
+
+    switch (operation)
+    {
+        
+        case "add":
+            result = parseNumberOne + parseNumberTwo;
+    break;
+        case "sub":
+            result = parseNumberOne - parseNumberTwo;
+    break;
+    case "mul":
+            result = parseNumberOne * parseNumberTwo;
+    break;
+         default:
+             return Results.BadRequest("Not valid operator!!!");
+             // Console.WriteLine("not valid operators");

```

Comment:
you can use `Alt+Shift+F` to format the code in Visual studio 😊 it helps with reviewing the code